src/TLuaInterpreter.cpp

```diff
@@ -15835,3 +15836,106 @@ int TLuaInterpreter::getMouseEvents(lua_State * L)
+{
+    QString key = getVerifiedString(L, __func__, 1, "key");
+    if (key.isEmpty()) {
+        return warnArgumentValue(L, __func__, "you must provide key");
+    }
+
+    auto success = [&]()
+    {
+        lua_pushboolean(L, true);
+        return 1;
+    };
+
+    Host& host = getHostFromLua(L);
+    if (host.mpMap && host.mpMap->mpMapper) {
+        if (key == "mapperRoomSize") {
+            host.mpMap->mpMapper->slot_setRoomSize(getVerifiedInt(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperExitSize") {
+            host.mpMap->mpMapper->slot_setExitSize(getVerifiedInt(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperRoundRooms") {
+            host.mpMap->mpMapper->slot_toggleRoundRooms(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperShowRoomIds") {
+            host.mpMap->mpMapper->slot_setShowRoomIds(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperShowMapInfo") {
+            host.mMapInfoContributors.insert(getVerifiedString(L, __func__, 2, "value"));
+            host.mpMap->mpMapper->slot_updateInfoContributors();
+            return success();
+        }
+        if (key == "mapperHideMapInfo") {
+            host.mMapInfoContributors.remove(getVerifiedString(L, __func__, 2, "value"));
+            host.mpMap->mpMapper->slot_updateInfoContributors();
+            return success();
+        }
+        if (key == "mapperShow3dView") {
+            host.mpMap->mpMapper->slot_toggle3DView(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperPanelVisible") {
+            host.mpMap->mpMapper->slot_setMapperPanelVisible(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperShowRoomBorders") {
+            host.mMapperShowRoomBorders = (getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "generalEnableGMCP") {
+            host.mEnableGMCP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "generalEnableMSDP") {
+            host.mEnableMSDP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "generalEnableMSSP") {
+            host.mEnableMSSP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "generalEnableMSP") {
+            host.mEnableMSP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "inputLineStrictUnixEndings") {
+            host.mUSE_UNIX_EOL = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "inputLineCommandSeparator") {
```

Comment:
Lets leave it out for now in order to get the PR through in a timely manner. See https://github.com/Mudlet/Mudlet/issues/3677 for discussion.